src/widgets/image/lv_image.c

```diff
@@ -540,6 +540,58 @@ int32_t lv_image_get_scale_y(lv_obj_t * obj)
     return img->scale_y;
 }
 
+int32_t lv_image_get_width(lv_obj_t * obj)
```

Comment:
We should avoid using names which conflict with `lv_obj_*` API names.as it would confuse bindings and our XML parser as well. 

 ```suggestion
int32_t lv_image_get_src_width(lv_obj_t * obj)
```

Reply:
OK do you have a proposed name you are happy with?